modules/n1ql/pages/n1ql-intro/sysinfo.adoc

```diff
@@ -47,11 +47,11 @@ It is analogous to a relational database table.
 
 Keyspaces::
 
-Keyspaces map to collections in Couchbase Server.
+Keyspaces map to collections in Couchbase Capella.
 A keyspace may be referred to using a full keyspace reference.
 For example, `default:{backtick}travel-sample{backtick}.inventory.airline` refers to the `airline` collection in the `inventory` scope in the `travel-sample` bucket.
 A keyspace may also be referred to using a partial keyspace reference, comprising just the collection name.
-In this case, the namespace, bucket, and scope must be implied by the current xref:queriesandresults.adoc#query-context[query context].
+In this case, the namespace, bucket, and scope must be implied by the current xref:n1ql-intro/queriesandresults.adoc#query-context[query context].
 +
 For compatibility with legacy versions of Couchbase Server, the name of the scope and collection may be omitted when referring to the default collection in the default scope within a bucket.
```

Comment:
I didn't touch instances of this line (`For compatibility with legacy versions of Couchbase Server...`) in the docs set, as I figured it wouldn't really make sense otherwise and it may contain useful information.